core/settings.py

```diff
@@ -259,6 +265,24 @@
 }
 
 
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.PyMemcacheCache',
+        'LOCATION': '127.0.0.1:11211',  # Default Memcached server address
+    }
+}
```

Comment:
```
Traceback (most recent call last):
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\User\Desktop\SIT378\hardhat-website\home\ratelimit_middleware.py", line 12, in __call__
    if cache.get('global_lockout'):
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\django\core\cache\backends\memcached.py", line 75, in get
    return self._cache.get(key, default)
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\pymemcache\client\hash.py", line 347, in get
    return self._run_cmd("get", key, default, default=default, **kwargs)
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\pymemcache\client\hash.py", line 322, in _run_cmd
    return self._safely_run_func(client, func, default_val, *args, **kwargs)
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\pymemcache\client\hash.py", line 211, in _safely_run_func
    result = func(*args, **kwargs)
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\pymemcache\client\base.py", line 687, in get
    return self._fetch_cmd(b"get", [key], False, key_prefix=self.key_prefix).get(
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\pymemcache\client\base.py", line 1133, in _fetch_cmd
    self._connect()
  File "C:\Users\User\Desktop\SIT378\hardhat-website\venv\lib\site-packages\pymemcache\client\base.py", line 424, in _connect
    sock.connect(sockaddr)

Exception Type: ConnectionRefusedError at /
Exception Value: [WinError 10061] No connection could be made because the target machine actively refused it
```